seedrandom.js

```diff
@@ -35,7 +35,7 @@ var global = this,
     significance = math.pow(2, digits),
     overflow = significance * 2,
     mask = width - 1,
-    nodecrypto;         // node.js crypto module, initialized at the bottom.
```

Comment:
I'd ideally like to keep seedrandom.js working as a standalone <script src="seedrandom.js"> browser script, which means require() will fail here.  Does your package.json change work without the change in seedrandom.js?